src/harness.ts

```diff
@@ -35,34 +36,51 @@ let harnessId = 0;
  * @param target the root DNode to search
  * @param key the key to match
  */
-function findDNodeByKey(target: DNode, key: string | object): HNode | WNode | undefined {
-	if (target && typeof target === 'object') {
-		if (target && typeof target === 'object' && target.properties && target.properties.key === key) {
-			return target;
-		}
+function findDNodeByKey(target: RenderResults, key: string | object): HNode | WNode | undefined {
+	if (!target) {
+		return;
+	}
+	if (target instanceof Array) {
 		let found: HNode | WNode | undefined;
-		if (target.children) {
-			target.children
-				.forEach((child) => {
-					if (found) {
-						if (findDNodeByKey(child, key)) {
-							console.warn(`Duplicate key of "${key}" found.`);
-						}
-					}
-					else {
-						found = findDNodeByKey(child, key);
-					}
-				});
-		}
+		target.forEach((node) => {
+			if (found) {
+				if (findDNodeByKey(node, key)) {
+					console.warn(`Duplicate key of "${key}" found.`);
+				}
+			}
+			else {
+				found = findDNodeByKey(node, key);
+			}
+		});
 		return found;
+	} else {
+		if (typeof target === 'object') {
```

Comment:
the code coverage is indicating an uncovered branch...  We are likely not testing a `string` value in here.